topics/ecology/tutorials/lotus2-identifying-fungi/tutorial.md

```diff
@@ -0,0 +1,462 @@
+>
+> Make sure you have the right version (2.32+galaxy0). You can check the version by clicking the {% icon tool-versions %} (blocks) icon.
+>
+> 1. {% tool [LotuS2](.bx.psu.edu/repos/earlhaminst/lotus2/lotus2/2.32+galaxy0) %} with the following parameters:
+>    - *"Single- or Paired-end data?"*: `Paired-end list`
+>        - In *"List of paired reads"*: choose the paired-list you created in the previous section: `Colombia ITS2 fastq pairs` (or whatever name you gave to the collection)
+>    - In *"Mapping file (optional)"*: `Colombia_ITS2_mapping.tsv`
+>    - Forward (and Reverse) Primer: Leave blank
+>    - *"Clustering algorithm"*: `VSEARCH`
+>    - *"Taxonomy aligner for taxonomic profiling of OTUs"*: `Lambda, LCA against custom reference database`
+>        - *"Taxonomy reference database"*: `Use a built-in taxonomy database`
+>            - *"Using reference database"*: `ITS fungi specific (UNITE)`
+>    - In *"Other Clustering Options"*:
+>        - *"Minimum size of dereplicated raw reads (optional)"*: `10:1,5:2,3:3`
+>    - In *"Other Taxonomy Options"*:
+>        - *"Amplicon type"*: `ITS2`
+>        - *"Tax group"*: `fungal 18S/23S/ITS annotation`
+>    - In *"Other options"*:
+>        - *"SDM option file"*: `sdm_miSeq_ITS.txt`
+>
+> 2. Once all the parameters above are entered, click the **"Run Tool"** button at the end.
+> 3. You should see a green box on the next page saying "Started tool LotuS2 and successfully added 1 job to the queue". The box lists the 6 outputs that the tool produces.
+> 4. This step can take 5-10 minutes to run or longer depending on how many other jobs are running on the usegalaxy server. While we are waiting for it to finish, you can do the next step on "Creating your own mapping.tsv" file
+> 5. You will know when the tool has finished, because all the outputs in the history will turn green.
+> 
+> 
+>    > <comment-title>Notes</comment-title>
+>    >
+>    > - Remember to choose _Paired-end list_ in the sequencing read data section. Galaxy will pick up the Paired-end list available in the History, which will have the name you gave it in the _Create a list of pairs_ step
+>    > - In *"Forward (and Reverse) Primer"*: Leave blank, as we have already provided them in the mapping.tsv file. If you are carrying out an analysis using your own data, you may add here the primer sequences used in your analyses in stead of in the mapping file, if you prefer.
+>    > - In *"Other Clustering Options"*: *"Minimum size of dereplicated raw reads (optional)"*: we put `10:1,5:2,3:3`. Each "X:Y" pair means "A unique dereplicated read must be seen at least X times in at least Y samples". So, if a sequence read is only found in 1 sample, it must be present in 10 copies. If a sequence read is found in only 2 samples, it must be found in 5 copies in each, etc. This is so that sequence errors are not taken as novel biological sequences, reducing the occurrences of false positive OTUs or ASVs
```

Comment:
Do you know of any source/studies where numbers for this are proposed ?

Reply:
No, but I will add a line that says why you might want to use more lenient values